lib/rails_api_format/lib/rails_api_format/error.rb

```diff
@@ -0,0 +1,25 @@
+module RailsApiFormat
+  class Error
+    include ActiveModel::Model
+    include ActiveModel::Serialization
+
+    ATTRIBBUTES = %i(status error validations)
+    attr_accessor(*ATTRIBBUTES)
```

Comment:
Fixed in https://github.com/fs/rails-base-api/commit/570f2cc77243c03a7548d7fb1046e8fbc02ea79c